bubblewrap.c

```diff
@@ -602,16 +670,13 @@ switch_to_user_with_privs (void)
 }
 
 static void
-drop_privs (void)
+drop_privs (bool keep_requested_caps)
 {
-  if (!is_privileged)
```

Comment:
I haven't tested it, though I suspect that this will cause https://pagure.io/releng/issue/6550 to regress; `systemd-nspawn` blocks `capset()`, but with this check removed, we'll always be calling it. (Note that https://github.com/projectatomic/bubblewrap/pull/122 originally worked around this by only setting caps in the setuid case).